molgenis-ontology-core/src/main/resources/ontologycore_en.properties

```diff
@@ -0,0 +1 @@
+O01=The nodePath cannot be null ''{0}''
```

Comment:
add dot at the end of the line

Reply:
assign

Reply:
proposal: do nothing, message on master branch doesn't have a dot at the end, updating messages is out of scope for the global exception handling feature.